### DISCUSSION TITLE
Citation database not showing up in "Export"

### DISCUSSION THREAD

**Discussion by William-mat** [Zettlr/Zettlr#3170] [Q&A]
I'm following the guide to setting up a reference database in Zettlr 2.1.3 for windows. I've followed the steps outlined here https://docs.zettlr.com/en/academic/citations/ 
but in step 3 when I go to the "Export" tab in preferences, the menu option for selecting my citation database is simply missing!

Does anybody have a good explanation/solution? Any help is much appreciated!

**Comment by nasteffe** [comment]
The screenshots on that page are from an earlier version of the software. The version that you (and I) are using has placed that field in the "Citations" tab to the right of the "Export" tab in preferences.

**Reply by nathanlesage** [reply]
Good catch! You are right, it's now in the Citations tab and the documentation is outdated there.

**Reply by William-mat** [reply]
Thanks, that solved it! Seems pretty obvious in hindsight, must have tried just about all the other tabs without noticing this one!
